package.json

```diff
@@ -30,7 +30,7 @@
     "uuid": "^3.3.2"
   },
   "scripts": {
-    "start": "chmod +x ./init.sh && ./init.sh && cp env-config.js ./public/ && npm run build:css && react-scripts start",
```

Comment:
How is this related to the ticket?

Reply:
@dkayiwa that script stops the application from running and removing it is the only way the application will start.

Reply:
Was that caused by changes in this ticket? If not, then it should be a separate ticket and pull request.

Reply:
@dkayiwa no it was not. There's a ticket that's been put up to solve the issue with the start script. Let me rebase.